packages/di/src/common/decorators/intercept.ts

```
@@ -50,7 +50,12 @@ export function bindIntercept(target: any, propertyKey: string | symbol, token:
       },
       next
     );
-  };
+  }
+
+  descriptor!.value = newMethod;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)